src/main/java/first/controllers/BaseHtmlTemplate.java

```diff
@@ -1,7 +1,8 @@
 package first.controllers;
 
-public interface StudentControllerInter {
-    String str = " <head>\n" +
+public interface BaseHtmlTemplate {
+
+    public String main = " <head>\n" +
```

Comment:
Because it is variable of interface it should named like MAIN - constant. Also, keyword public is extra because variables of interface have public static by default.